.kitchen.dokken.yml

```diff
@@ -1,6 +1,6 @@
 driver:
   name: dokken
-  chef_version: latest
```

Comment:
either runit or mongodb breaks with Chef13.  Not disagreeing, just that this is what was required to get the dependencies to converge.